docs/PayPal-commercetools-workflow.postman_collection.json

```diff
@@ -88,20 +88,20 @@
                   "tests[\"Status code \" + responseCode.code] = responseCode.code === 200 || responseCode.code === 201;",
                   "var data = JSON.parse(responseBody);",
                   "if(data.results && data.results[0] && data.results[0].id && data.results[0].version){",
```

Comment:
okay, this does not work. After using the request, the parameters product-id,... should be set. You need to adjust the "tests" of that search to make it work